src/controller/config/controller.go

```diff
@@ -89,37 +101,26 @@ func (c *controller) UpdateUserConfigs(ctx context.Context, conf map[string]inte
 
 func (c *controller) validateCfg(ctx context.Context, cfgs map[string]interface{}) (bool, error) {
 	mgr := config.GetCfgManager(ctx)
-	flag, err := c.authModeCanBeModified(ctx)
-	if err != nil {
-		return true, err
-	}
-	if !flag {
-		if failedKeys := c.checkUnmodifiable(ctx, cfgs, common.AUTHMode); len(failedKeys) > 0 {
+	if c.authModeUpdated(ctx, cfgs) {
+		flag, err := c.authModeCanBeModified(ctx)
+		if err != nil {
+			return false, err
```

Comment:
and the first return value is always false

Reply:
Updated